apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DynamicWorkloadActivity.java

```diff
@@ -51,6 +51,11 @@ public class DynamicWorkloadActivity extends TestOutputActivityBase {
     // By default, set high workload to 70 voices, which is reasonable for most devices.
     public static final double WORKLOAD_PROGRESS_FOR_70_VOICES = 0.53;
 
+    public static final String KEY_USE_ADPF = "use_adpf";
+    public static final boolean VALUE_DEFAULT_USE_ADPF = false;
+    public static final String KEY_SCROLL_GRAPHICS = "scroll_graphics";
+    public static final boolean VALUE_DEFAULT_SCROLL_GRAPHICS = false;
```

Comment:
Please update https://github.com/google/oboe/blob/2de755200046c4821ca5fd220713837a05bb024a/apps/OboeTester/docs/AutomatedTesting.md?plain=1#L91

Reply:
I like to update the docs after the code is merged.
Otherwise people may try out the docs and it doesn't work.